src/odemis/driver/spectrapro.py

```diff
@@ -200,7 +200,7 @@ def __init__(self, name, role, port, turret=None, calib=None,
 
         # for now, it's fixed (and it's unlikely to be useful to allow less than the max)
         max_speed = 1000e-9 / 10 # about 1000 nm takes 10s => max speed in m/s
-        self.speed = model.MultiSpeedVA(max_speed, range=[max_speed, max_speed], unit="m/s",
+        self.speed = model.MultiSpeedVA({"wavelength": max_speed}, range=[max_speed, max_speed], unit="m/s",
```

Comment:
I think the range is set incorrectly the min and maximum is the same now.
I would either expect "range=[0, max_speed]' or something a like

Reply:
Actually it's read-only, so it's not possible to be changed. That's why all the values are the same. That said, I think the name is weird. It should just be "speed".